lms/templates/video.html

```diff
@@ -26,6 +26,9 @@ <h3 class="hd hd-2">${display_name}</h3>
           <div class="video-player">
               <div id="${id}"></div>
               <h4 class="hd hd-4 video-error is-hidden">${_('No playable video sources found.')}</h4>
+              <h4 class="hd hd-4 video-hls-error is-hidden">
+                  ${_('This browser can not play HLS videos. Try using a different browser, such as Google Chrome.')}
```

Comment:
I would say "Your browser does not support this video format. Try using a different browser." (I wouldn't mention Chrome specifically in case something changes in the future.)